pkg/apis/configuration/v1beta1/kongconsumergroup_types.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2021 Kong, Inc.
```

Comment:
Do we have instructions on how to write the Copyright lines in comments 
 for license? `Copyright 2021` seems weird here because we actually added this file since 2023.

Reply:
Yeah nice catch, copy-paste error. But I wonder maybe some day in separeate PR gets rid of  those types of headers/comments from the entire repo. They do not bring value IMHO